exporter/awskinesisexporter/internal/compress/noop_compression.go

```diff
@@ -9,12 +9,6 @@ type noop struct {
 	data io.Writer
 }
 
-func NewNoopCompressor() Compressor {
```

Comment:
Is there any reason to remove the method here?

Reply:
the compressor abstraction got removed